### PR TITLE
Add common operations section to User Guide.

### PR DESCRIPTION
**Pull request by Naarcha-AWS**
### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by Naarcha-AWS** [comment]
@IanHoang: Let me know if this is what you had in mind.

**Review by IanHoang — APPROVED** [review]

File: _benchmark/user-guide/understanding-workloads/common-operations.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+layout: default
+title: Common operations
+nav_order: 16
+grand_parent: User guide
+parent: Understanding workloads
+---
+
+# Common operations
+
+[Test procedures]({{site.url}}{{site.baseurl}}/benchmark/user-guide/understanding-workloads/anatomy-of-a-workload#_operations-and-_test-procedures) use a variety of operations, found inside the `operations` directory of a workload. This page details the most common operations found inside Benchmark workloads.
```
Comment (vagimeli) [inline _benchmark/user-guide/understanding-workloads/common-operations.md:11]: Should "OpenSearch" precede "Benchmark to be consistent naming convention throughout this doc (for example, line 123?)

**Review by vagimeli — APPROVED** [review]
Well done. LGTM.

File: _benchmark/user-guide/understanding-workloads/common-operations.md (hunk 1 above)
Comment (Naarcha-AWS) [reply _benchmark/user-guide/understanding-workloads/common-operations.md:11]: Yes.

Comment (Naarcha-AWS) [inline _benchmark/user-guide/understanding-workloads/common-operations.md:11]:
```suggestion
[Test procedures]({{site.url}}{{site.baseurl}}/benchmark/user-guide/understanding-workloads/anatomy-of-a-workload#_operations-and-_test-procedures) use a variety of operations, found inside the `operations` directory of a workload. This page details the most common operations found inside OpenSearch Benchmark workloads.
```

**Pull request merged**